proto/spacemesh/v1/smesher_types.proto

```diff
@@ -128,3 +128,17 @@ message PostSetupStatus {
     PostSetupOpts opts = 3; // setup options previously set by the user
     string error_message = 4; // The error message, if the state is STATE_ERROR
 }
+
+message SmeshingStatusRequest {
+    string smesher_id = 1;
+    repeated uint32 epochs = 2; // Epochs to query status for. If empty, returns status for current, next and subsequent.
+}
+
+message SmeshingStatusResponse {
+    string status = 1;
+    uint32 epoch = 2;
+    uint64 estimated = 3;
+    string reason = 4;
+    string processed_bytes = 5;
```

Comment:
How is this different than `PostSetupStatus`? Which bytes are being counted here?

Reply:
this is for ui inform view. like show 
```
 processedBytes:  "Processed 176 GB of 256 GB"
 processedPercentage: "86% completed"
```

Reply:
What do these bytes represent? I don't understand

Also the data passed from the backend to the frontend should be numeric (i.e., not strings). The frontend can interpret and display that data in different ways (e.g., using different languages)